official/wide_deep/README.md

```diff
@@ -0,0 +1,135 @@
+# Predicting Income with the Census Income Dataset
+
+There are two samples provided in this directory. Both allow you to move from
+single-worker training to distributed training without any code changes, and
+make it easy to export model binaries for prediction, but with the following
+distinction:
+
+
+* The sample provided in [Low Level TensorFlow](./tflowlevel) uses the low level
```

Comment:
Should we rename this directory to tf_core or something?

Reply:
It was tfcore but @itsmeolivia asked that we change.